Jetcaster/mobile/src/main/java/com/example/jetcaster/ui/home/Home.kt

```diff
@@ -332,13 +327,15 @@ private fun HomeScreenReady(
 
     Surface {
         val podcastUri = navigator.currentDestination?.content
-        val showGrid = homeState.showGrid(navigator.scaffoldValue)
         if (podcastUri.isNullOrEmpty()) {
-            HomeScreen(
-                homeState = homeState,
-                showGrid = showGrid,
-                modifier = Modifier.fillMaxSize()
-            )
+            CompositionLocalProvider(
+                LocalAnimatedVisibilityScope provides animatedContentScope
+            ) {
```

Comment:
Whats the reasoning for moving this further down the tree?

Reply:
I noticed an issue where the animation was being executed in other areas when the AnimationScope was propagated to the SupportingPane. To address this, I modified it so that the scope is passed only to the node where the animation should occur.

https://github.com/user-attachments/assets/55c3a2a1-0556-4c2c-b433-bd1c3ccfef4c